packages/server/__tests__/api/grants.test.js

```diff
@@ -143,6 +156,10 @@ describe('`/api/grants` endpoint', () => {
                 const badResponse = await fetchApi(`/grants/${assignedEndpoint}`, agencies.offLimits, fetchOptions.admin);
                 expect(badResponse.statusText).to.equal('Forbidden');
             });
+            it('includes assigned by information for the grant to an agency', async () => {
+                expect(json.find((a) => a.assigned_by_name === assignedBy.NV1.assigned_by_name)).to.be.ok;
+                expect(json.find((a) => a.assigned_by_name === assignedBy.NV2.assigned_by_name)).to.be.ok;
```

Comment:
Seems worth testing the other fields are present too. Can we verify that name, email, and avatar color are all as expected?

Reply:
Done here as well!